### PR TITLE
PL-146: Browser Extension

### PR DESCRIPTION
**Pull request by tonesija**
Adds a browser extension which will:
 - check if there is a pdf opened
 - inject a button/popup into the dom of the default reader
 - clicking on a button will redirect to our reader with the url of the pdf as a http query

DONE:
 - pdf detection
 - button which takes you to our reader 
TODO:
 - change our welcome screen to detect if there is a url query and load the pdf from the url
 - injected button/popup style
 - style for the extension

### CODE REVIEW

File: browser-extension/manifest.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "manifest_version": 3,
+  "name": "PDF - Legacy",
+  "version": "1.0.0",
+  "description": "Detects and opens pdfs in the PDF - Legacy reader",
```
Comment (matteovisotto) [inline browser-extension/manifest.json:5]: I would write:
"Detects and opens PDFs in PDFL (PDF - Legacy) reader"

File: browser-extension/pdfPopup.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+function generateOpenWithPDFLButton() {
+  var btn = document.createElement("button");
+  var t = document.createTextNode("Open in PDF - Legacy");
```
Comment (matteovisotto) [inline browser-extension/pdfPopup.js:3]: Maybe PDFL? People loves abbreviations

File: browser-extension/popup.html [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    <link rel="icon" href="assets/icon.png" />
+    <link rel="stylesheet" href="style.css" />
+
+    <title>PDF - Legacy</title>
+  </head>
+
+  <body>
+    <h1>PDF - Legacy Extension</h1>
+
+    <h4 id="text">Open pdfs with a distinguished reader</h4>
```
Comment (matteovisotto) [inline browser-extension/popup.html:17]: PDFs (it is better to see PDF in capital letters and the s small)

**Review by matteovisotto — COMMENTED** [review]
Just two personal comments that can be ignored, the rest is fine

**Comment by tonesija** [comment]
> Just two personal comments that can be ignored, the rest is fine

The html is all just a placeholder, designer are free to change everything.

**Review by gabrielekarra — APPROVED** [review]
+1 from me, there are some empty line but who cares, test it! let's gooo

**Comment by NinaGnjidic** [comment]
Tested locally (throws CORS but well test it), code is generally okay, +1 from me

**Pull request merged**